rmgpy/thermo/convertTest.py

```diff
@@ -102,7 +102,7 @@ def test_convert_Wilhoit_to_NASA(self):
             self.assertAlmostEqual(Cp_nasa, Cp_wilhoit, delta=1e0)
             H_wilhoit = wilhoit.getEnthalpy(T)
             H_nasa = nasa.getEnthalpy(T)
-            self.assertAlmostEqual(H_nasa, H_wilhoit, delta=2e1)
```

Comment:
How does adding a pair of independent functions affect the accuracy of converting Wilhoit to NASA?

Reply:
If we use the typical way to modify entropy of a NASA object, it is converted to Wilhoit and then reconverted back into NASA. There is some error in the fitting procedure from Wilhoit to NASA (and vise versa). This error is small compared to most things in RMG, but for isotope studies, it is significant to not allow proper enrichments to be derived. This method is meant to modify the NASA object without converting to Wilhoit (thereby reducing the error). To answer your question, this method does not affect the accuracy of converting between Wilhoit and NASA, it does around the whole issue by keeping objects as NASA.

Reply:
I understand what they are intended to do, but where do these functions interface with the conversion method?  Since you had to define them in the pxd file I assume the methods did not exist before you made them?  and they aren't called except by each other anywhere in the changes here?  Which logically implies to me that they aren't called by existing RMG functions/objects?  Are you missing a commit where you modify the toNASA function?

Reply:
Good questions. These methods are never intended to interface with the conversion methods (like toNASA or toWilhoit).

Currently they are not called anywhere in the RMG code, since they are not necessary in a typical RMG generation (to my knowledge). I use these methods in the isotope module, which there will be a pull request for soon. I could have just written separate methods in the isotope module to perform this action, but I thought that would be better object oriented design to have a NASA method to modify these values since it doesn't necessarily involve anything related to isotopes. (plus, someone else might find modifying NASA polynomials without introducing extra error useful too). 

If there something I should rewrite in the PR that could make this clearer, let me know.

Reply:
Ok, that makes sense, but if this doesn't interface with toNASA why does this require changes in test_convert_Wilhoit_to_NASA?

Reply:
That is a very good question, sorry I misunderstood the previous two attempts. 

That question baffled me when I first wrote this commit, hence the comment at the end of the commit. My best guess is that the scipy.optimize function behaved differently when the cythonized object of NASA was modified. The nonlinear fitting within the `toNASA` function could have found a different minima. 

For some reason, after updating to most recent master, and trying out the lower values, the lower values worked (which is rebased into the current branch). I think there might be a potential for finding different minima in the test method.